static/index.html

```diff
@@ -27,7 +33,7 @@
 <body>
     <h1>Partify</h1>
     <p>
-        Every emoji should be a party
+        Because every emoji should be a party
```

Comment:
I like this the way it is
```suggestion
        Every emoji should be a party
```